pkg/cluster/check/kubernetes.go

```diff
@@ -231,14 +231,20 @@ func K8sPodReadyAssertion(ctx context.Context, cluster cluster.K8sProvider, name
 	var notReadyPods []string
 
 	for _, pod := range pods.Items {
+		ready := false
```

Comment:
this check had a bug - if pod has no `PodReady` condition, it declared pod as ready